src/Team-Services-Bot.Api.UnitTests/Team-Services-Bot.Api.UnitTests.csproj

```diff
@@ -33,6 +33,7 @@
     <CodeAnalysisRuleSet>..\Common\TestCodeRules.ruleset</CodeAnalysisRuleSet>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <DocumentationFile>bin\Debug\Vsar.TSBot.UnitTests.xml</DocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
Do we really need global suppression? Also, warning should be suppressed for Release configuration as well.